doc/contributing.asciidoc

```diff
@@ -85,6 +85,24 @@ git format-patch origin/master <1>
 <1> Replace `master` by the branch your work was based on, e.g.,
 `origin/develop`.
 
+[[python]]
+Running Development Changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+To run the git instance of qutebrowser with modifications (useful for testing out small changes)
+run the following commands in the qutebrowser directory:
+
+----
+python3 -m qutebrowser --temp-basedir --no-err-windows
```

Comment:
I think I've said this elsewhere (but can't find it right now): I'd prefer just suggesting `--debug` over `--no-err-windows`. It shows some nice debug logs, and the error windows are IMHO more comfortable than getting a dump of stuff in the terminal.

Reply:
Oops forgot to push those changes